lib/condorManager.py

```diff
@@ -56,105 +91,182 @@ def condorSubmitOne(
     cluster_or_uname, schedd_name=None, pool_name=None, schedd_lookup_cache=condorMonitor.local_schedd_cache
 ):
+    """Release a single job from hold in the queue.
+
+    Args:
+        cluster_or_uname (str): The ClusterId or username of the job to release.
+        schedd_name (str, optional): The name of the schedd. Defaults to None.
+        pool_name (str, optional): The name of the pool. Defaults to None.
+        schedd_lookup_cache (optional): The cache for schedd lookups. Defaults to condorMonitor.local_schedd_cache.
+
+    Returns:
+        str: The output of the condor_release command.
+    """
     opts = "%s " % cluster_or_uname
     return cached_exe_cmd("condor_release", opts, schedd_name, pool_name, schedd_lookup_cache)
 
 
-##############################################
-#
-# Issue a condor_reschedule
-#
 def condorReschedule(schedd_name=None, pool_name=None, schedd_lookup_cache=condorMonitor.local_schedd_cache):
+    """Issue a condor_reschedule command.
+
+    Args:
+        schedd_name (str, optional): The name of the schedd. Defaults to None.
+        pool_name (str, optional): The name of the pool. Defaults to None.
+        schedd_lookup_cache (optional): The cache for schedd lookups. Defaults to condorMonitor.local_schedd_cache.
+    """
     cached_exe_cmd("condor_reschedule", "", schedd_name, pool_name, schedd_lookup_cache)
     return
 
 
 ##############################################
 # Helper functions of condorAdvertise
 def usetcp2str(use_tcp):
+    """Convert use_tcp flag to a string suitable for the Condor command line.
+
+    Helper functions of `condorAdvertise`.
+
+    Args:
+        use_tcp (bool): If True, use TCP.
+
+    Returns:
+        str: The use_tcp flag formatted for the command line.
+    """
     if use_tcp:
         return "-tcp "
     else:
         return ""
 
 
 def ismulti2str(is_multi):
+    """Convert is_multi flag to a string suitable for the Condor command line.
+
+    Helper functions of `condorAdvertise`.
+
+    Args:
+        is_multi (bool): If True, indicate multiple.
+
+    Returns:
+        str: The is_multi flag formatted for the command line.
+    """
     if is_multi:
         return "-multiple "
     else:
         return ""
 
 
-##############################################
-#
-# Remove a job from the queue
-#
+#############################################
+# HTCondor Advertise function
 def condorAdvertise(classad_fname, command, use_tcp=False, is_multi=False, pool_name=None):
+    """Advertise a HTCondor ClassAd to the collector.
+
+    Args:
+        classad_fname (str): The filename of the classad.
+        command (str): The Condor command to advertise.
+        use_tcp (bool, optional): If True, use TCP. Defaults to False.
+        is_multi (bool, optional): If True, indicate multiple. Defaults to False.
+        pool_name (str, optional): The name of the pool. Defaults to None.
+
+    Returns:
+        str: The output of the condor_advertise command.
```

Comment:
Should `condor_advertise` be enclosed in ` ?

Reply:
It could be either way, it is not in the code. Anyway done